Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -86,9 +89,53 @@ protected override void OnExecute(object parameter)
 
             foreach (var filename in _openFileDialog.FileNames)
             {
-                using (var components = project.VBComponents)
+                var extension = filename.Split('.').Last();
+
+                var sourceText = string.Join(Environment.NewLine, File.ReadAllLines(filename));
+
+                string updatedModuleText;
+
+                var result = VBACodeStringParser.Parse(sourceText, t => t.startRule());
+
+                var tempHelper = (CodeExplorerItemViewModel)parameter;
+                var updatedFolder = (parameter is CodeExplorerCustomFolderViewModel) ? tempHelper.Name : tempHelper.GetSelectedDeclaration().CustomFolder;
+
+                const string folderAnnotation = "'@Folder";
```

Comment:
per `VBAParserAnnotationFactory`, this should probably be `AnnotationType.Folder.ToString()` (and thus not `const`).